### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by bbrozyna**
New issue templates for Bug and feature requests

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```
Comment (micdol) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]: None of these steps are relevant to our use case. This clearly is some ui stuff. Copy pasting huh?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
[the remark this answers is not in the document]
Comment (bbrozyna) [reply .github/ISSUE_TEMPLATE/bug_report.md]: good idea

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (bbrozyna) [reply .github/ISSUE_TEMPLATE/bug_report.md]: we are writing UI app, don't you know that? :p

**Review by bbrozyna — COMMENTED** [review]
yes

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (micdol) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: `feature` or `improvement` label?`

Comment (bbrozyna) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: I see ` enhancement ` is available

**Review by micdol — APPROVED** [review]

**Pull request merged**